release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -82,6 +82,10 @@ Deploying a three-node cluster is supported on Nutanix as of {product-title} {pr
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-three-node.adoc#installing-nutanix-three-node[Installing a three-node cluster on Nutanix].
 
+[id="ocp-4-14-installation-gcp-confidential-vms"]
+==== Installing a cluster on GCP using Confidential VMs is generally available
+In {product-title} {product-version}, using Confidential VMs when installing your cluster is generally available. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-gcp-enabling-confidential-vms_installing-gcp-customizations[Enabling Confidential VMs].
```

Comment:
Any PR to remove the IMPORTANT note in the linked page, which tells the "Confidential Computing is a Technology Preview feature only"? Thanks!

Reply:
Handled in this PR: https://github.com/openshift/openshift-docs/pull/64086

Reply:
Thanks, Ben! Sorry @jianli-wei that this effort was split across two writers, I know it makes tracking more annoying :sweat_smile: